elastic/apm-server
# [apmpackage] Remove duplicate input variable attributes

## Motivation/summary
Starting with elastic/kibana#106535 Kibana manages most of the
input variable attributes. Remove all attributes from the manifest
except for the name, type, multi and default value.

Non-functional change


## Checklist



~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~
~- [ ] Documentation has been updated~

## How to test these changes
* checkout `apm-server` from github
* checkout the `package-registry` from github, and run `mage build`, create a `0.4.0` folder inside `build/package-storage/packages/apm/` and copy `apm-server/apmpackage/apm/*` (from the apm-server repo) into the package-registry folder `build/package-storage/packages/apm/0.4.0/` and run  `go run .`
* from inside the `apm-integration-testing` start the test setup and point to the local registry `./scripts/compose.py start master --no-apm-server --with-elastic-agent --package-registry-url=http://host.docker.internal:8080`
* add an apm integration to a policy, check that default values are there, that the created policy looks ok and check that the integration can be edited afterwards.
